source/introduction/what-is-ethereum.rst

```diff
@@ -1,42 +1,74 @@
+and developers to decide what it should be used for.
 
-The Ethereum platform itself is featureless or value-agnostic. Similar to programming languages, it is up to entrepreneurs and developers to decide what it should be used for. However, it is clear that certain application types benefit more than others from Ethereum's capabilities. Specifically, ethereum is **suited for applications that automate direct interaction between peers or facilitate coordinated group action across a network**. For instance, applications for coordinating peer-to-peer marketplaces, or the automation of complex financial contracts. Bitcoin allows for individuals to exchange cash without involving any middlemen like financial institutions, banks, or governments. Ethereum’s impact may be more far-reaching. In theory, financial interactions or exchanges of any complexity could be carried out automatically and reliably using code running on Ethereum. Beyond financial applications, any environments where trust, security, and permanence are important – for instance, asset-registries, voting, governance, and the internet of things – could be massively impacted by the Ethereum platform.
+Specifically, Ethereum is suited for applications that automate direct interaction
+between peers or facilitate coordinated group action across a network. Examples
+include applications for coordinating peer-to-peer marketplaces or the
+automation of complex financial contracts.
+
+In contrast with blockchain technologies that largely limit themselves to the exchange of money,
+Ethereum serves much wider purposes. Beyond financial applications, any environments
+where trust, security, and permanence are important – for instance, asset-registries,
+voting, governance, and the internet of things – make ideal candidates for the Ethereum platform.
 
 ================================================================================
 How does Ethereum work?
 ================================================================================
 
-Ethereum incorporates many features and technologies that will be familiar to users of Bitcoin, while also introducing many modifications and innovations of its own.
+Ethereum incorporates many features and technologies that will be familiar to users
+of other blockchain technologies while also introducing many innovations of its own.
+
+For example, :ref:`Ethereum's blockchain holds accounts <Accounts>` and tracks their state.
+All state transitions on the Ethereum blockchain are transfers of value and information between accounts.
 
-Whereas the Bitcoin blockchain was purely a list of transactions, :ref:`Ethereum's basic unit is the account <Accounts>`. The Ethereum blockchain tracks the state of every account, and all state transitions on the Ethereum blockchain are transfers of value and information between accounts. There are two types of accounts:
+There are two types of accounts:
```

Comment:
Yes, I think I agree.

It is better to just describe the features and functionality, rather than just saying "Bitcoin + x, y, z".    That assumes knowledge of Bitcoin which people may or may not have.